mne_bids_pipeline/steps/preprocessing/_01_data_quality.py

```diff
@@ -62,74 +72,72 @@ def assess_data_quality(
     subject: str,
     session: Optional[str],
     run: Optional[str],
+    task: Optional[str],
     in_files: dict,
-) -> None:
+) -> dict:
     """Assess data quality and find and mark bad channels."""
     import matplotlib.pyplot as plt
 
     out_files = dict()
-    orig_run = run
-    # raw_ref_run will be .pop()ed inside this loop, do not include it
-    raw_keys = list(key for key in in_files.keys() if key != "raw_ref_run")
-    for key in raw_keys:
-        bids_path_in = in_files.pop(key)
-        if key == "raw_noise":
-            run = "noise"
-        elif key == "raw_rest":
-            run = "rest"
-        else:  # raw_run-{run}
-            run = orig_run
-        if _do_mf_autobad(cfg=cfg):
-            if key == "raw_noise":
-                bids_path_ref_in = in_files.pop("raw_ref_run")
-            else:
-                bids_path_ref_in = None
-            auto_scores = _find_bads_maxwell(
-                cfg=cfg,
-                exec_params=exec_params,
-                bids_path_in=bids_path_in,
-                bids_path_ref_in=bids_path_ref_in,
-                key=key,
-                subject=subject,
-                session=session,
-                run=run,
-                out_files=out_files,
-            )
+    key = f"raw_task-{task}_run-{run}"
```

Comment:
And keys for the "data file being processed" are now standardized this way, as `raw_task-{task}_run-{run}`. The one exception is the reference run gets stored as `raw_ref_run` still, because it's *in addition to / a companion of the data file being processed*, and which run/task it's from is not relevant